tests/serverpod_test_server/lib/src/custom_classes.dart

```diff
@@ -31,15 +31,14 @@ class CustomClass2 {
   dynamic toJson() => {'text': value};
 }
 
-class CustomClass3 implements SerializableModel, ProtocolSerialization {
+class CustomClass3 implements SerializableModel {
   final String value;
 
   CustomClass3(this.value);
 
   @override
   String toJson() => value;
 
-  @override
   String toJsonForProtocol() => value;
```

Comment:
Suggestion: Remove toJsonForProtocol implementation if the interface is not implemented by this class.

We should have examples of both used as fields.